coding-concepts.js

```diff
@@ -8,36 +8,42 @@
 // --------------------1) What will this log?
 
 const cohort = "Bravo 2023"
-// console.log(cohort.split(""))
+console.log(cohort.split(""))
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: {"B", "r", "a", "v", "o"," ","2","0","0","3"} 
```

Comment:
Right on! Nice job accounting for the space, just make sure we use [] for an array.